src/site/content/en/blog/coop-coep/index.md

```diff
@@ -104,6 +106,11 @@ clearer, let's define them first:
 
 ## Deploy COOP and COEP to make your website cross-origin isolated
 
+{% Aside 'gotchas' %}
```

Comment:
```suggestion
{% Aside %}
```

Reply:
`gotchas` are for "a common problem that the reader wouldn't know without specialized knowledge of the topic", this is just additional information so it should be a regular aside. https://web.dev/handbook/web-dev-components/#asides